packages/nouns-api/src/graphql/resolvers/PropLotListResolvers.ts

```diff
@@ -135,6 +138,14 @@ const resolvers: IResolvers = {
         type: FilterType.MultiSelect,
         label: 'Tags',
         options: [
+          {
```

Comment:
I added a comment in the code a few lines above this but i'd probably load the tags from the DB here. then iterate over them and build out these options using that data.

I'll comment it here but to support this i'd maybe add a displayName property to your tag model and you can load those in here for the label. Then i'd replace the hardcoded `'REQUEST'` values with the TagType property.

Would endup looking something like

```
 const tags: Tag[] = prisma.tags.findMany(...)
 
 const filterOptions = tags.map(tag => {
  return {
     id: `${FILTER_IDS.TAG}-${tag.type}`
     label: tag.displayName,
     value: buildFilterParam(FILTER_IDS.TAG, tag.type),
     selected: Boolean(
         root.tagParams?.includes(buildFilterParam(FILTER_IDS.TAG, tag.type)),
     ),
  }
})

const tagFilter: PropLotFilter = {
 ....
 options: filterOptions,
}
```

Reply:
Yea, I like the way this looks.

Reply:
On the point about tags like Hot I'd probably create a map for these "virtual tags" and then load them in here. Something like this?

```
// load + build DB tags

const virtualTags = {
  HOT: {
    type: HOT,
    displayName: "Hot",
    filterFn: (ideas) => { ... logic to filter ideas }, // not sure about this property
  },
  ...
}; // Maybe store these in a standalone file and import them here - virtualTags.ts?

 const virtualFilterOptions = Object.keys(virtualTags).map(vT => {
  return {
     id: `${FILTER_IDS.TAG}-${virtualTags[vT].type}`
     label: tag.displayName,
     value: buildFilterParam(FILTER_IDS.TAG, virtualTags[vT].type),
     selected: Boolean(
         root.tagParams?.includes(buildFilterParam(FILTER_IDS.TAG, virtualTags[vT].type)),
     ),
  }
})


const tagFilter: PropLotFilter = {
 ....
 options: [...filterOptions, ...virtualFilterOptions],
}
```